libraries/csharp-net22-buildtime/context/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+dotnet restore
+dotnet build
```

Comment:
do you know if there's a way to set a mirror for dotnet packages or if that's just something users set in their project file?

Reply:
you can define it in the csproj file, yea you can also use private repos and whatnot. the main package mgmt system is managed here by the `__ALGO__.csproj` file.